src/posts/xarray-biology/index.md

```diff
@@ -0,0 +1,148 @@
+---
+title: 'Accelerating Biological Analysis with Xarray: Why You Should Try Using It Today'
+date: '2025-05-04'
+authors:
+  - name: Ian Hunt-Isaak
+    github: ianhi
+summary: 'A discussion of how Xarray fits into Biological analysis workflows'
+---
+
+I'm Ian, a multimodal microscopist, and the new "Xarray Community Developer." In this role, funded by the Chan Zuckerberg Institute, I work full-time to support the usage of Xarray in biological and biomedical applications.
+
+I ended up here because, five years in graduate school, I was sitting at my computer frustratedly trying to keep track of microscopy image metadata as I passed images through complicated, larger-than-memory jobs on the cluster. Which axis was what? Where did the exposure times go? I know I changed the media at 32 minutes, but what time index was that? Desperate for a solution, I found `Xarray`, seemingly a solution to all my woes. But I left thinking it was not for me as the examples were geoscience-centric. Fortunately, I eventually rediscovered `Xarray`, mentally translated the examples to biology, and found it incredibly useful. It is so useful that it since served as my default tool for all analyses in both structural and microbiology.
+
+My story is not unique. You have likely experienced similar challenges when working with array data (e.g., microscopy images, genomic sequences, or anything else you might currently analyze using `NumPy`). This post is for you whether you have heard of `Xarray` before or not.
+
+In this post, I will introduce the concepts of `Xarray` at a high level with biological context and give examples where it is already in use. Then, based on interviews I conducted with scientists and scientific software developers from across biology I will explain what has limited adoption. Finally, I will describe what we (Biologists and Xarray contributors) can do to increase adoption.
```

Comment:
I suggest rephrasing this to start with a short description of what you've down over the past two months, and present this post as a summary.

```suggestion
I have spent the last XXXXX.... 

In this post, I will introduce Xarray at a high level with biological context and give examples where it is already in use. Then, based on interviews I conducted with scientists and scientific software developers from a number of biology subfields I will explain what has limited adoption. Finally, I will describe what we (biologists and Xarray contributors) can do to increase adoption.
```